Sources/DistributedCluster/Pattern/WorkerPool.swift

```diff
@@ -172,21 +216,12 @@ extension WorkerPool {
     /// E.g. the `dynamic` mode uses a receptionist key, and will add any workers which register
     /// using this key to the pool. On the other hand, static configurations could restrict the set
     /// of members to be statically provided etc.
-    public struct Selector {
-        enum _Selector {
-            // TODO: let awaitAtLeast: Int // before starting to direct traffic
-            case dynamic(DistributedReception.Key<Worker>)
-            case `static`([Worker])
-        }
-
-        let underlying: _Selector
```

Comment:
@ktoso I wonder why `underlying` is needed and not to use enum directly? I've updated it to be enum, but maybe there are some concerns...

Reply:
Don't do public enums -- we can never add a case to them basically because of source compatibility. So this has to remain as a struct that hides the enum inside it

Reply:
ah, good point, haven't thought about this.